docs/changelog-fragments/576.doc.rst

```diff
@@ -0,0 +1,3 @@
+Fixed example for exec_command -- by :user:`pbrezina`
+
+The example now contains correct method code and is fully functional.
```

Comment:
```suggestion
Fixed the example of invoking remote commands by using
``Channel.exec_command()`` in snippets -- by :user:`pbrezina`.

Its previously showcased version wasn't as robust.
```

Reply:
I wouldn't say robust, it simply was not functional.

Reply:
@pbrezina was it nonfunctional unconditionally, then?

Reply:
Yes.